lib/extras/packed_image.h

```diff
@@ -244,9 +244,12 @@ class PackedPixelFile {
   std::vector<PackedExtraChannel> extra_channels_info;
 
   // Color information of the decoded pixels.
-  // If the icc is empty, the JxlColorEncoding should be used instead.
-  std::vector<uint8_t> icc;
+  enum {
+    kPreferColorEncoding,
+    kPreferIcc
+  } preferred_color_profile = kPreferColorEncoding;
```

Comment:
Could you add a comment here to explain what this toggle means / how it is used in enc/dec?

I'm not sure if `preferred_color_profile` is the best name for this variable, maybe something like `primary_colorspace_description` could make sense, with the other one being a 'secondary' colorspace description which is supposed to approximate the primary colorspace description, but it's the primary one that is authoritative and takes precedence, and the secondary one is only used to maximize compatibility and/or in cases where the format does not support signaling the primary one.

Reply:
Thanks, something like this?

Reply:
LGTM!